src/main/java/no/fintlabs/template/AdresseLinjeTemplateService.java

```diff
@@ -0,0 +1,29 @@
+package no.fintlabs.template;
+
+import no.fintlabs.template.model.ElementConfig;
+import no.fintlabs.template.model.ObjectTemplate;
+import no.fintlabs.template.model.ValueTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AdresseLinjeTemplateService {
+
+    public ObjectTemplate createTemplate() {
+        return ObjectTemplate
+                .builder()
+                .addTemplate(
+                        ElementConfig
+                                .builder()
+                                .key("adresselinje")
```

Comment:
litt usikker på key her vs den i adressetemplateservice

Reply:
Se b426296 :)

Reply:
Jepps, ser det var noen mangler her og der, men da skal det være good to go? :D